cypress/integration/brewery/RenameScenario.spec.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import utils from '../../commons/TestUtils';
+import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
+import { Scenarios, ScenarioManager, ScenarioParameters, Login } from '../../commons/actions';
+
+describe('Create scenario and rename it', () => {
+  Cypress.Keyboard.defaults({
+    keystrokeDelay: 0,
+  });
+
+  const SCENARIO_DATASET = DATASET.BREWERY_ADT;
+  const SCENARIO_RUN_TEMPLATE = RUN_TEMPLATE.BASIC_TYPES;
+
+  function forgeScenarioName() {
+    const prefix = 'Scenario to rename - ';
+    const randomString = utils.randomStr(7);
+    return prefix + randomString;
+  }
+
+  const anotherScenario = forgeScenarioName();
+  let anotherScenarioId;
+  const scenarioNamesToDelete = [];
+  scenarioNamesToDelete.push(anotherScenario);
+
+  before(() => {
+    Login.login();
+    Scenarios.createScenario(anotherScenario, true, SCENARIO_DATASET, SCENARIO_RUN_TEMPLATE).then((data) => {
+      anotherScenarioId = data.scenarioCreatedId;
+    });
+  });
+
+  beforeEach(() => {
+    Login.relogin();
+  });
+
+  after(() => {
+    ScenarioManager.deleteScenarioList(scenarioNamesToDelete);
+  });
+
+  it('Rename scenarios several times and launch', () => {
+    const scenarioNamePrefix = forgeScenarioName();
+    const scenarioName = scenarioNamePrefix + '_A';
+    let newScenarioName = scenarioNamePrefix + '_B';
+
+    Scenarios.createScenario(scenarioName, true, SCENARIO_DATASET, SCENARIO_RUN_TEMPLATE).then((data) => {
+      const scenarioId = data.scenarioCreatedId;
+      ScenarioManager.switchToScenarioManager();
+      ScenarioManager.getScenarioEditableLabel(scenarioId)
+        .click()
+        .type('{selectAll}{backspace}' + newScenarioName + '{esc}')
```

Comment:
Could you use the `renameScenario` action with `newScenarioName + '{esc}'` as a parameter ?

Reply:
It's not possible because the rename function already contain _{enter}_ type and it should cause conflict with another key passed as parameter.